schedule/migration/s390x-zVM-Media-Upgrade.yaml

```diff
@@ -0,0 +1,25 @@
+name:           s390x-zVM-Media-Upgrade.yaml
+description:    >
+    This is for s390x zVM media upgrade migration.
+schedule:
+  - installation/bootloader_s390
+  - installation/welcome
+  - installation/disk_activation
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/partitioning
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
+  - installation/select_patterns
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - boot/reconnect_mgmt_console
+  - installation/first_boot
+  - console/scc_deregistration
```

Comment:
We shouldn't include the scc_deregistration here, it is not following migration flow.

You can try below:
  - console/system_prepare
  - console/hostname
  - console/force_scheduled_tasks
  - update/patch_sle
  - shutdown/grub_set_bootargs
  - shutdown/cleanup_before_shutdown
  - shutdown/shutdown

Refer to https://openqa.nue.suse.com/tests/11164534#settings

Reply:
Fixed.